include/kernels/dslash_mdw_fused.cuh

```diff
@@ -311,22 +311,22 @@ namespace quda {
 
         if (Arg::type == MdwfFusedDslashType::D4_D5INV_D5PRE) {
           if (arg.small_kappa) {
-            construct_matrix_a_d5<M_sm, false, Arg>(arg, sm_a); // dagger = false
+            construct_matrix_a_d5<M_sm, false, Arg>(arg, sm_a, arg.alpha, arg.beta); // dagger = false
```

Comment:
What is this change to `construct_matrix_a_mg5inv` motivated by (e.g., the addition of explicit alpha and beta arguments)?

Reply:
In most cases `alpha = arg.alpha`, `beta = arg.beta`; but in a few others cases we need to use other values for `alpha` and `beta`. Previously this is done by modifying `arg.alpha` and `arg.beta`, which is a bad practice in the first place. Now that arg becomes const so we need to change the function interface to pass in `alpha` and `beta` explicitly.